client/js/components/AccountLoginForm/AccountLoginForm.js

```diff
@@ -0,0 +1,147 @@
+import React, {Component} from 'react';
+import {Form, Field} from 'react-final-form';
+import {
+  Image,
+  TouchableOpacity,
+  View,
+  Text,
+  TextInput,
+  ImageBackground,
+} from 'react-native';
+import EmailIcon from '../../assets/signinicons/EmailIcon.png';
+import PasswordIcon from '../../assets/signinicons/PasswordIcon.png';
+import styles from './styles';
+import MainSigninButton from '../MainSignupButton/MainSignupButton';
+import {Mutation} from '@apollo/react-components';
+import ApolloClient from 'apollo-boost';
+import gql from 'graphql-tag';
+import {withNavigation} from 'react-navigation';
+import InactiveButton from '../../assets/buttons/Inactivespacebutton.png';
+import {apolloClientAddress} from '../../config/constant';
+
+const LOGIN_MUTATION = gql`
+  mutation login($email: String!, $password: String!) {
+    login(email: $email, password: $password) {
+      token
+      user {
+        id
+        name
+        email
+      }
+    }
+  }
+`;
+
+class AccountLoginForm extends Component {
+  render() {
+    return (
+      <Mutation
+        mutation={LOGIN_MUTATION}
+        client={
+          new ApolloClient({
+            uri: apolloClientAddress,
+          })
+        }>
+        {login => (
+          <View style={styles.accountLoginContainer}>
+            <Form
+              onSubmit={async values => {
+                try {
+                  const data = await login({
+                    variables: {
+                      email: values.email,
+                      password: values.password,
+                    },
+
+                    // todo add navigate to home page.
+                  });
+                  console.log(data);
```

Comment:
remove all the unnecessary console logs from your project.

Reply:
Done on this file and all other extra console.logs